src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -79,6 +98,29 @@ def extract_strings(self) -> Mapping[int, Set[str]]:
                 )
                 continue
 
+    @metrics.wraps("process_messages.limit_messages")
+    def limit_messages(self, keys_to_remove: Sequence[PartitionIdxOffset]) -> None:
```

Comment:
While we use this for rate limiting, its behavior is simply a filter. 
Just call it filter.

Reply:
done